2-ui/99-ui-misc/03-event-loop/article.md

```diff
@@ -9,15 +9,15 @@ In this chapter we first cover theoretical details about how things work, and th
 
 ## Event Loop
 
-The concept of *event loop* is very simple. There's an endless loop, when JavaScript engine waits for tasks, executes them and then sleeps waiting for more tasks.
+The *event loop* concept is very simple. There's an endless loop, when the JavaScript engine waits for tasks, executes them and then sleeps, waiting for more tasks.
```

Comment:
I think it should be 'where' instead of 'when'.

when the JavaScript engine waits for tasks ->  where the JavaScript engine waits for tasks